packages/core/src/app/address/AddressForm.tsx

```diff
@@ -202,12 +202,18 @@ class AddressForm extends Component<AddressFormProps & WithLanguageProps> {
         const address = mapToAddress(place, countries);
 
         forIn(address, (value, fieldName) => {
+            if (fieldName === AUTOCOMPLETE_FIELD_NAME && value === undefined) {
+                return;
+            }
```

Comment:
Can you explain what the problem would be if `setFieldValue(AUTOCOMPLETE_FIELD_NAME, undefined);` runs?

Reply:
I am not sure when that will run, since if address1 is undefined we fallback to the existing implementation.

Reply:
I'm asking because I see it's an early return to avoid `setFieldValue(AUTOCOMPLETE_FIELD_NAME, undefined);`.

Perhaps we just don't want to run `onChange(AUTOCOMPLETE_FIELD_NAME, undefined);`?